website/static/playground.js

```diff
@@ -439,6 +439,23 @@ function getExample(parser) {
         "  }",
         "}"
       ].join("\n");
+    case "json":
+      // Excerpted & adapted from Wikipedia, under the Creative Commons Attribution-ShareAlike License
```

Comment:
Could you add a link to the source?

Reply:
A link to the license?

Reply:
@skube I think @ikatyang wanted a link to the source of the JSON document.

Reply:
Sorry I misunderstood. I added a link to source now.